src/main/features/dashboard/views/macro/claimStatus/claimPartAdmitted.njk

```diff
@@ -0,0 +1,102 @@
+{% from "externalLink.njk" import externalLink %}
+
+{% macro claimPartAdmittedAlreadyPaidForDefendantDashboard(claim) %}
+  {{ t('We’ve emailed {{ defendantName }} telling them when and how you said you paid the claim.',
+    { defendantName: claim.claimData.defendant.name }) }}
+{% endmacro %}
+
+{% macro claimPartAdmittedAlreadyPaidForClaimantDashboard(claim) %}
+  {{ t('The defendant believes that they’ve paid the claim in full.') }}
+{% endmacro %}
+
+{% macro claimPartAdmittedAlreadyPaidForDefendantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('Your response to the claim') }}</h2>
+  <p>
+    {{ t('We’ve emailed {{ claimantName }} telling them when and how you said you paid the claim.',
+      { claimantName: claim.claimData.claimant.name }) }}
+  </p>
+  <p>
+    {{ t('We’ll contact you to let you know how they respond. They can confirm you’ve paid and the claim is ' +
+      'settled, or they can proceed with it.') }}
+  </p>
+  <p>
+    {{ _downloadResponseLink(claim) }}
+  </p>
+{% endmacro %}
+
+{% macro claimPartAdmittedAlreadyPaidForClaimantClaimDetails(claim) %}
+  {% if claim.response.paymentDeclaration.paidDate %}
```

Comment:
Remove, when method is used paid date is always required, otherwise it should fail.